fedimint-server/src/net/api.rs

```diff
@@ -264,21 +264,55 @@ impl ConsensusApi {
     }
 
     pub async fn get_consensus_status(&self) -> ApiResult<ConsensusStatus> {
-        let our_last_contribution = self.get_epoch_count().await;
+        let peers_connection_status = self.peer_status_channels.get_all_status().await;
         let latest_contribution_by_peer = self.latest_contribution_by_peer.read().await.clone();
-        let peers_connection_status: HashMap<PeerId, anyhow::Result<PeerConnectionStatus>> =
-            self.peer_status_channels.get_all_status().await;
-        // How much time we consider a contribution recent for a "grace time".
-        // For instance, even if a peer isn't connected right now, if it contributed
-        // recently then we won't flag it.
-        const MAX_DURATION_FOR_RECENT_CONTRIBUTION: Duration = Duration::from_secs(60);
-
-        Ok(calculate_consensus_status(
-            latest_contribution_by_peer,
-            our_last_contribution,
-            peers_connection_status,
-            MAX_DURATION_FOR_RECENT_CONTRIBUTION,
-        ))
+        let epoch_count = self.get_epoch_count().await;
+
+        let status_by_peer = peers_connection_status
+            .into_iter()
+            .map(|(peer, connection_status)| {
+                let last_contribution = latest_contribution_by_peer.get(&peer).cloned();
+                let flagged = last_contribution.unwrap_or(0) + 100 < epoch_count;
```

Comment:
100 is a lot. it doesn't make sense for HBBFT. Does it make sense for Aleph?

Reply:
No for aleph it would be one as a session is already 5 min. What number do you prefer?

Reply:
I switched it to 10, does that work?

Reply:
On previous version this was zero. But I think one is okay.